src/booking.js

```diff
@@ -0,0 +1,13 @@
+class Booking {
+  constructor(bookingData) {
+    this.id = bookingData.id // STRING!!!
+    this.customerID = bookingData.userID;
+    this.customer;
+    this.date = bookingData.date;
+    this.roomNumber = bookingData.roomNumber;
+    this.price = bookingData.price;
```

Comment:
some where down the line you may want to retrieve `roomDetails` for each booking as well